src/module-2/components/stylesheets/filters.css

```diff
@@ -0,0 +1,64 @@
+.filters{
```

Comment:
try to locale `.filters` styles and other repeating styles in one file, otherwise it will be hard to track and debug styles

Reply:
[Modified](https://github.com/front-camp-2021/html-css-module-Kyanka/commit/a96e851a8ea308063ab34f162ef68b4714e0db1e)